app/views/posts/index.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Posts#index</h1>
-<h2>This is the posts index page</h2>
\ No newline at end of file
+ <div class="main-container">
+ <div class="users-wrapper flex">
+    <div class="img-cont">
+      <%= image_tag @user.photo, :size => "50x50", alt: "user photo" %>
+    </div>
+    <div class="user-details">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+ </div>
+
+ <% @posts.each do |post| %>
+    <div class="posts-enclosure">
+   <div class="users-post">
+       <h2><%= link_to "Post title: #{post.title}", user_post_path(@user.id, post.id), class: "link-button" %></h2>
+       <p><%= post.text %></p>
+       <h4>Comments: <%=post.comments_counter %>, Likes: <%= post.likes_counter %></h4>
+   </div>
```

Comment:
- [ Required ] Good job implementing some changes from the previous reviewer. Since some parts of the app weren't working during the first review please consider making the design of this view similar to the design on the project wireframe. This page should list all the posts for a given user and five recent comments per post. Please see the screenshot below for the sections that are missing from your design.

## Screenshot ( The page should have listed 6 posts that I added to the database)
![Screenshot from 2023-06-16 02-06-37](https://github.com/Abe1able/Bloggingly/assets/16392046/efb0be4a-31f9-4957-b3a3-c2ebc32d4835)

## Project wireframe
![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)